spec/uploader/configuration_spec.rb

```diff
@@ -116,7 +116,11 @@
     end
 
     it "adds a convenient in-class setter" do
-      expect(uploader_class.foo_bar).to eq('foo')
+      uploader_class.foo_bar('bar')
+      expect(uploader_class.foo_bar).to eq('bar')
+
+      uploader_class.foo_bar(false)
+      expect(uploader_class.foo_bar).to eq(false)
```

Comment:
Please create a separate test case for this part. The implementation itself looks good!

Reply:
Sorry, that's fixed :) bcc3bb0
Should I squash commits?

Reply:
That's not necessary!